server/src/utils/getConfig.ts

```diff
@@ -14,12 +14,54 @@ import {
   VV_WAITING_TITLE_ENV_NAME,
   VV_LOGO_URL_ENV_NAME,
   VV_POSTCALL_SURVEY_TYPE_ENV_NAME,
-  VV_POSTCALL_SURVEY_OPTIONS_SURVEYURL_ENV_NAME
+  VV_POSTCALL_SURVEY_OPTIONS_SURVEYURL_ENV_NAME,
+  VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_PROMPT_ENV_NAME,
+  VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_TYPE_ENV_NAME,
+  VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_TITLE_ENV_NAME,
+  VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_SAVE_BUTTON_TEXT_ENV_NAME
 } from '../constants';
 
-import { ServerConfigModel, ClientConfigModel, PostCallSurveyType, PostCallConfig } from '../models/configModel';
+import {
+  ServerConfigModel,
+  ClientConfigModel,
+  PostCallSurveyType,
+  PostCallConfig,
+  MSFormsSurveyOptions,
+  OneQuestionPollOptions,
+  OneQuestionPollType,
+  CustomSurveyOptions
+} from '../models/configModel';
 import { getDefaultConfig } from './getDefaultConfig';
 
+export const getMSFormsOptions = (defaultConfig: ServerConfigModel): MSFormsSurveyOptions => {
+  const options: MSFormsSurveyOptions = defaultConfig.postCall?.survey?.options as MSFormsSurveyOptions;
+  const postcallSurveyUrl = process.env[VV_POSTCALL_SURVEY_OPTIONS_SURVEYURL_ENV_NAME] ?? options.surveyUrl;
+  return { surveyUrl: postcallSurveyUrl };
+};
+
+export const getCustomSurveyOptions = (defaultConfig: ServerConfigModel): CustomSurveyOptions => {
+  const options: CustomSurveyOptions = defaultConfig.postCall?.survey?.options as CustomSurveyOptions;
+  const postcallSurveyUrl = process.env[VV_POSTCALL_SURVEY_OPTIONS_SURVEYURL_ENV_NAME] ?? options.surveyUrl;
+  return { surveyUrl: postcallSurveyUrl };
+};
+
+export const getOneQuestionPollOptions = (defaultConfig: ServerConfigModel): OneQuestionPollOptions => {
+  const options: OneQuestionPollOptions = defaultConfig.postCall?.survey?.options as OneQuestionPollOptions;
+  const surveyTitle = process.env[VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_TITLE_ENV_NAME] ?? options.title;
+  const surveyPromptType: OneQuestionPollType =
+    (process.env[VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_TYPE_ENV_NAME] as OneQuestionPollType) ??
+    (options.pollType as OneQuestionPollType);
+  const surveyPrompt = process.env[VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_PROMPT_ENV_NAME] ?? options.prompt;
+  const surveySaveButtonText =
+    process.env[VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_SAVE_BUTTON_TEXT_ENV_NAME] ?? options.saveButtonText;
+  return {
+    title: surveyTitle,
+    prompt: surveyPrompt,
+    pollType: surveyPromptType,
+    saveButtonText: surveySaveButtonText
```

Comment:
Missing `answerPlaceholder` here

Reply:
My bad, missed adding stuff related to the answerPlaceholder. Fixed that now.